src/glmfit.jl

```diff
@@ -67,6 +76,14 @@ function updatemu!{T<:FPVector}(r::GlmResp{T}, linPr::T)
     r
 end
 
+"""
+    wrkresp(r::GlmResp)
+
+Working response.
```

Comment:
Again, the style guide says the description should start with an imperative verb. So:
```
Return the working response for GLM response `r`.
```